### ISSUE TITLE
EnvVar not recognised

### ISSUE THREAD

**Issue by Dimenjineer** [allquixotic/encuum#15]
I followed the guide for this (windows 10) and the program seems to fail at the first hurdle, not reading the variables in .env properly.
I tried different naming conventions: config.env, configuration.env or just .env all have the same result.
It's also not creating a log file, which I assume is due to it failing before it can read the log_level var so I can't submit a log file via the recommended route. Here's the console dump: 

D:\Enjin backup\encuum-master>cargo run --release
    Finished release [optimized] target(s) in 0.41s
     Running `target\release\encuum.exe`
thread 'main' panicked at 'database_file must be set: EnvVar(NotPresent)', src\main.rs:49:45
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
error: process didn't exit successfully: `target\release\encuum.exe` (exit code: 101)

My .env reads:
email=********@hotmail.co.uk
password=******
website=www.theclandestine.enjin.com\forum
database_file=cland.db
keep_going=false
do_images=true
log_level=TRACE
sanitize_log=true
log_file=ErrorLog.log

I also tried different naming conventions for the database_file and found no difference with or without quotation marks or file extension .db
Rearranging the order of the variables seems to result in it failing to read a different variable, with the same result.

**Comment by allquixotic** [comment]
This is probably a duplicate of https://github.com/allquixotic/encuum/issues/10 .

**Comment by allquixotic** [comment]
For starters, don't put anything but the domain in the `website` variable. In your case, that would just be `www.theclandestine.enjin.com` -- no `\forum`.

Not sure if that'll fix it, but it might be weirding out on the backslash anyway.

**Comment by Dimenjineer** [comment]
Thank you so much for your help. I'm not sure why it was quibbling about EnvVar when the website was the issue, but after some messing around I found that I had to remove both the \forum and the starting www. to get it to start (the documentation gives an example specifically using www. so you might want to check what's going on there).

It's now running enough to get an error log out of it so I will hopefully be able to get it running with some fiddling with the variables.
